src/styles/bookListAndInput.scss

```diff
@@ -1,11 +1,11 @@
 @media only screen and (min-width: 1600px) {
   div.bookList-bookInput-box {
-    border: 2px solid black;
     display: flex;
     flex-direction: column;
     gap: 2.5rem;
     align-items: center;
     padding-top: 2.375rem;
+    padding-bottom: 16.563rem;
 
```

Comment:
- [ ] Please kindly move the logo to the right to match the original design 🙃 
- [ ] And fix the distance between the logo and the link `BOOKS`, and the distance between `BOOKS` and `CATEGORIES` 🙏🏾 
- [ ] Make sure the buttons `comment, remove and Edit` match exactly to the original design and is not in bold, and the author title as well 🙃 
- [ ] Make sure the spam `Completed` match the design as well 😁 

Your design

<img width="862" alt="Screenshot 2023-06-15 at 21 40 00" src="https://github.com/momo-87/book-store/assets/74501523/256ba362-6eda-4bf6-bf7b-c6a478febedf">

Original design 

<img width="901" alt="Screenshot 2023-06-15 at 21 48 19" src="https://github.com/momo-87/book-store/assets/74501523/01715cb3-8b40-4b11-8ad1-0b361cb61ecd">